pages/search.js

```diff
@@ -88,21 +88,13 @@ export default function Search({ users }) {
       </Head>
       <Page>
         <h1 className="text-4xl mb-4 font-bold">Search</h1>
-
-        <div className="relative">
-          <input
-            placeholder="Search users"
-            className="border-2 hover:border-orange-600 transition-all duration-250 ease-linear rounded px-6 py-2 mb-4 block w-full"
-            name="keyword"
-            value={inputValue}
-            onChange={(e) => setInputValue(e.target.value)}
-          />
-          {filteredUsers && (
-            <div className="absolute inline-block top-0 right-0 bottom-auto left-auto translate-x-1/4 -translate-y-1/3 rotate-0 skew-x-0 skew-y-0 scale-x-100 scale-y-100 py-1 px-1.5 text-xs leading-none text-center whitespace-nowrap align-baseline font-bold bg-orange-600 text-black rounded-full z-10">
-              {filteredUsers.length}
```

Comment:
Thank you for your contribution 👍 

Not sure why this functionality has been removed?

Reply:
It might be better to update your fork and then make the changes again, if you prefer to do it that way, you can close this pull request and then make a new one